app.rb

```diff
@@ -0,0 +1,31 @@
+require 'sinatra/base'
+require './models/peeps'
+require './models/accounts'
+
+class Chitter < Sinatra::Base
+
+  get '/' do
```

Comment:
the route names could be made RESTful (from rubric): GET /peeps
GET /peeps/:id
GET /peeps/new
GET /peeps/update
POST /peeps
POST /peeps/:id/update
POST /peeps/:id/delete